File: 27/remove-element.java
class Solution {
    public int removeElement(int[] nums, int val) {
        int N = nums.length;
        for(int i=0; i<N; i++){
            if(nums[i]==val) {
                nums[i] = nums[N-1];
                N--;
                i--;
            }
        }
        return N;
    }
}

File: 27/remove-element.c
int removeElement(int* nums, int numsSize, int val){
    int len = 0;
    for(int i=0; i<numsSize; i++){
        if(nums[i]==val){
        }else{
            nums[len] = nums[i];
            len++;
        }
    }
    return len;
}
